scripts/quick-helm-install.sh

```diff
@@ -58,18 +58,18 @@ kubectl -n stackrox exec deploy/central -- roxctl --insecure-skip-tls-verify \
   --password "${STACKROX_ADMIN_PASSWORD}" \
   central init-bundles generate stackrox-init-bundle --output - 1> stackrox-init-bundle.yaml
 
-installflags=("-f stackrox-init-bundle.yaml" "--set clusterName=\"my-secured-cluster\"")
-if $SMALL_INSTALL; then
-    installflags+=("--set sensor.resources.requests.memory=500Mi")
-    installflags+=("--set sensor.resources.requests.cpu=500m")
-    installflags+=("--set sensor.resources.limits.memory=500Mi")
-    installflags+=("--set sensor.resources.limits.cpu=500m")
+installflags=('-f' 'stackrox-init-bundle.yaml' '--set' 'clusterName=\"my-secured-cluster\"')
```

Comment:
No longer need to escape `\"` when inside of  single quotes

Reply:
True